codespell_lib/data/dictionary.txt

```diff
@@ -5681,6 +5681,8 @@ briliant->brilliant
 brilinear->bilinear
 brillant->brilliant
 brimestone->brimstone
+brin->bring
+brining->bringing
```

Comment:
`brining` is a common word in cooking. https://en.wikipedia.org/wiki/Brining
Either move this to rare, or delete it.

Reply:
Done in 57dbbf5